src/main/java/teammates/ui/webapi/action/ActionFactory.java

```diff
@@ -48,6 +48,9 @@
         map(ResourceURIs.INSTRUCTORS, DELETE, DeleteInstructorAction.class);
         map(ResourceURIs.INSTRUCTOR, GET, GetInstructorAction.class);
         map(ResourceURIs.INSTRUCTOR_PRIVILEGE, GET, GetInstructorPrivilegeAction.class);
+        map(ResourceURIs.RESPONSE_COMMENT, POST, CreateFeedbackResponseCommentAction.class);
+        map(ResourceURIs.RESPONSE_COMMENT, PUT, SaveFeedbackResponseCommentAction.class);
```

Comment:
Should be `UpdateFeedbackResponseCommentAction` now LOL. Take your time to change hahahah.